2.4/Dockerfile

```diff
@@ -38,22 +38,29 @@ RUN yum install -y epel-release && \
     # Loosen permission bits to avoid problems running container with arbitrary UID
     chmod -R a+rwx /opt/rh/mongodb24/root/var/lib/mongodb
 
-COPY run-mongod.sh /usr/local/bin/
-COPY contrib /var/lib/mongodb/
-
 # Loosen permission bits to avoid problems running container with arbitrary UID
 RUN chmod -R g+rwx /var/lib/mongodb
 
+# Get prefix path and path to scripts rather than hard-code them in scripts
+ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
+    MONGODB_PREFIX=/opt/rh/mongodb24/root/usr \
```

Comment:
@omron93 do you remember why do we have this env var and what it does? I see it was added in 3.2, then replicated here, but it is not referenced anywhere outside of the Dockerfiles.